data/role/jenkins-slave.yaml

```diff
@@ -183,3 +184,20 @@ windows_networks:
     local_ports: '8080'
     remote_ports: "*"
     enabled: true
+psick::windows::envs::envs_hash:
+  'AWS_ENV':
+    value: "%{::ec2_tag_env}"
+    mergemode: clobber
+    user: 'Administrator'
```

Comment:
@olegy-ait @MaximSternberg Small issue with the naming here, the variable is "PREFIX" but the EC2 tag is "region". Prefix includes env and region, right? So I think the EC2 tag should be called prefix too, and region should be a new tag and variable. What do you think?